src/software/thunderscope/gl/gl_widget.py

```diff
@@ -321,9 +321,11 @@ def set_camera_view(self, camera_view: CameraView) -> None:
         self.gl_view_widget.reset()
         if camera_view == CameraView.ORTHOGRAPHIC:
             self.gl_view_widget.setCameraPosition(
-                pos=pg.Vector(0, 0, 0), distance=1100, elevation=90, azimuth=-90
+                pos=pg.Vector(0, 0, 0),
+                distance=self.calc_orthographic_distance(),
+                elevation=90,
+                azimuth=-90,
             )
-            self.gl_view_widget.setCameraParams(fov=1.0)
```

Comment:
This line should remain in order for `calc_orthographic_distance` to make sense.